includes/csharp10-templates.md

```diff
@@ -1,10 +1,10 @@
 ---
 author: billwagner
 ms.author: wiwagn
-ms.date: 08/20/2021
+ms.date: 09/02/2021
 ms.topic: include
 ---
 > [!IMPORTANT]
 > The C# templates for .NET 6 use *top level statements*. Your application may not match the code in this article, if you've already upgraded to the .NET 6 previews. For more information see the article on [New C# templates generate top level statements](~/docs/core/tutorials/top-level-templates.md)
 >
-> The .NET 6.0 SDK also adds a set of *implicit* `global using` directives based on the workload of the project. Those implicit global usings include the most common namespaces for that project type. The implicit using depend on the workload. For more information see the article on [Breaking change: Implicit namespaces in C# projects](~/docs/core/compatibility/sdk/6.0/implicit-namespaces.md#new-behavior).
+> For certain C# project templates, the .NET 6 SDK also adds a set of *implicit* `global using` directives based on the workload of the project. Those implicit global `using` directives include the most common namespaces for that project type.
```

Comment:
We should probably use "SDK" here instead of "workload" for consistency. (I'll look in the other C# articles for other instances):
```suggestion
> The .NET 6 SDK also adds a set of *implicit* `global using` directives for projects that use the following SDKs:
>
> - Microsoft.NET.Sdk
> - Microsoft.NET.Sdk.Web
> - Microsoft.NET.Sdk.Worker.
> 
> Those implicit global `using` directives include the most common namespaces for those projects.
```